test/public/defaults.js

```diff
@@ -470,3 +475,46 @@ module.exports.waitForTableDataReload = (page, triggerFunction) => Promise.all([
     page.waitForSelector('table .atom-spinner'),
     triggerFunction(),
 ]).then(() => page.waitForSelector('table .atom-spinner', { hidden: true }));
+
+/**
+ * Tests whether sorting of main table by column with given id works properly
+ * It required there are a least two rows in the table
+ * @param {puppeteer.Page} page the puppeteer page
+ * @param {string} columnId subject column id
+ * @return {Promise<void>} promise
+ */
+module.exports.testTableAscendingSortingByColumn = async (page, columnId) => {
```

Comment:
Very nice idea. I have just 2 remarks:
- You just forgot a few `names` in comments
- I would go further, and also test the descending order and the going back to no order. Doing so, please carefully tailor the timeouts for the tests not to last hours...

Reply:
applied